ui_framework/src/global_styling/mixins/_popover.scss

```diff
@@ -0,0 +1,5 @@
+@mixin kuiPopoverTitle {
```

Comment:
This is not a global style. It is simply a part of popovers.

Reply:
Where should this go? It isn't available to the context menu if it's put inside of the /popover directory.

Reply:
I would say that if their functionality is so different that we can't make them a single component, the CSS should be separate as well so they can differ as they need. I just don't want us getting in the habit of putting things in global that aren't actually global (meaning, something that can be used against ANY component).

Right now looking at what we have in global, it's stuff that can be used anywhere.

Shadow, sizing, animation, colors, zindexs, typography....etc.

This one is only useful in two places and would never get reused outside of a popover. Having the same code in two spots is probably fine. Possibly a better way to handle it would be to `@import popover/_mixin.sass` or something into context_menu if it's a hard requirement.

Have any other ideas?

Reply:
Importing popover mixins is fine. I assume we'll want to do this with the kuiLink mixin as well since by your logic it also doesn't belong in `global_styling`?

Reply:
Eh. I just looked and we aren't using link in reset (though it looks like we mostly are, just copy/pasted). I think that would say it's a pretty good reason to stay if we want.

Don't care too much about that one. Could go either way.